src/commands/m1.ts

```diff
@@ -0,0 +1,17 @@
+import { Command } from '../types';
+
+const m1: Command = {
+  name: 'm1',
+  description: 'Apple silicon m1',
+  args: false,
+  cooldown: 10,
+  execute(msg) {
+    return msg.channel.send([
+      `👨‍💻 ***Czy Apple Silicon m1 jest gotowe dla developerów?*** 👩‍💻 \n`,
+      `https://isapplesiliconready.com/for/developer`,
```

Comment:
Nie jestem w 100% pewien, ale tutaj powinien być raczej jeden string a nie tablica. Jeżeli chcesz dać nową linię, to po prostu `.join("\n"` 😉

Reply:
Tablica powinna być ok

Reply:
![Screenshot 2020-12-13 at 00 25 52](https://user-images.githubusercontent.com/24244872/101999093-c4aa5880-3cd9-11eb-8242-25883d274bcb.png)

Może być tak.

Reply:
Ok, jest git 😄